test/amd-test.js

```diff
@@ -0,0 +1,25 @@
+(function(define) {
+    'use strict';
+
+    define(function() {
+        return {
+            test: function() {
```

Comment:
Could this function be imported instead?  It's a bit unsatisfying to think that we'd need to write a test in two places at all times.